dll/win32/syssetup/wizard.c

```diff
@@ -2968,6 +2968,26 @@ ProcessSetupInf(
     }
 #endif
 
+    WCHAR szData[MAX_PATH] = L"";
```

Comment:
Here and in what follows, you can re-use the (already correctly sized) `szValue` buffer that already exists.
```suggestion
    *szValue = UNICODE_NULL;
```